client/app/components/calendar/event/details/EventExpandableDescription.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { MAX_DESCRIPTION_LENGTH } from 'helpers/EventHelper';
+
+export default class EventExpandableDescription extends React.Component {
+  static propTypes = {
+    description: React.PropTypes.string
+  };
+
+  constructor(...args) {
+    super(...args);
+    this.state = { fullDescription: false };
+
+    this._handleDescriptionLength = this._handleDescriptionLength.bind(this);
```

Comment:
I think `toggleFullDescription` would be a better name than `handleDescriptionLength`